core/src/scene/styleContext.cpp

```diff
@@ -274,7 +274,7 @@ bool StyleContext::parseStyleResult(StyleParamKey _key, StyleParam::Value& _val)
             case StyleParamKey::outline_width:
             case StyleParamKey::font_stroke_width: {
                 double v = duk_get_number(m_ctx, -1);
-                _val = static_cast<float>(v);
+                _val = StyleParam::Width{static_cast<float>(v)};
```

Comment:
In `StyleParam`, `font_stroke_width` is set as a `float`, here it's set as a `StyleParam::Width`